enoslib/ansible/roles/utils/templates/ips.txt.j2

```diff
@@ -4,7 +4,7 @@
   all_ipv4_addresses:
 {{hostvars[host]['ansible_all_ipv4_addresses'] | to_nice_yaml(2) | indent(width=4, indentfirst=true)}}
   devices:
-{% for enos_device in enos_devices %}
+{% for enos_device in hostvars[host]['enos_devices'] %}
```

Comment:
I'm trying to undestand what was wrong here ? Could you help me ?

Reply:
When I used it with [this conf](https://github.com/Marie-Donnie/juice/blob/af598ca419171a00b4d33e733356badf649c2e0f/conf.yaml), I had only eth0 used for the control node and eth0 and eth1 on database nodes. For some reason, when using 'enos_devices', it always used only eth0 (I'm guessing it used 'enos_devices' from the control_node and never changed it afterwards) for every nodes.
As a result, when I wanted to add latency over eth1 using the generated file, there was never any eth1 written, only eth0.
This is actually what was used in the [host template](https://github.com/BeyondTheClouds/enoslib/blob/master/enoslib/ansible/roles/utils/templates/hosts.txt.j2#L2), and using this put all the interfaces used in the generated file.

Reply:
Got it, most probably due to https://github.com/BeyondTheClouds/enoslib/blob/master/enoslib/ansible/roles/utils/tasks/tc_ips.yml#L6-L7